packages/core/src/components/axes/axis.ts

```diff
@@ -76,6 +77,8 @@ export class Axis extends Component {
 			const timeLocale = Tools.getProperty(options, "locale", "time");
```

Comment:
Would
```suggestion
			const timeLocale = Tools.getProperty(options, "locale", "time") | englishLocale;
```
make it so we don't need the `if-else` block  and just call `timeFormatDefaultLocale(timeLocale)`?

Reply:
I don't think that change is doing what you think it's doing 😉 `|` is a bitwise operator ... also if/else is way more readable than an inline `||` IMO